packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/AddFont/FontPicker/index.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+  FontManager,
+  Font,
+  FONT_FAMILY_DEFAULT,
+  OPTIONS_DEFAULTS,
+} from '@samuelmeuli/font-manager';
+import React, { useState, useEffect, useRef } from 'react';
+import OutsideClickHandler from 'react-outside-click-handler';
+import { Portal } from 'reakit';
+import { SelectedFont } from './elements';
+import { FontList } from './List';
+
+export const FontPicker = ({
+  activeFontFamily = FONT_FAMILY_DEFAULT,
+  onChange,
+  apiKey,
+}) => {
+  const [expanded, setExpanded] = useState(false);
+  const [loadingStatus, setLoadingStatus] = useState('loading');
+  const [fontManager, setFontManager] = useState();
+  const [style, setStyle] = useState({ x: 0, y: 0 });
+  const ref = useRef(null);
+
+  useEffect(() => {
+    const manager = new FontManager(
+      apiKey,
+      activeFontFamily,
+      { ...OPTIONS_DEFAULTS, limit: 200 },
+      onChange
+    );
+    setFontManager(manager);
+
+    manager
+      .init()
+      .then(() => setLoadingStatus('finished'))
+      .catch((err: Error) => {
+        setLoadingStatus('error');
+        console.error(err);
+      });
+    // eslint-disable-next-line
+  }, []);
+
+  useEffect(() => {
+    if (ref && ref.current) {
+      const styles = ref.current.getBoundingClientRect();
+      setStyle({
+        x: styles.x,
+        y: styles.y,
+      });
+    }
+  }, [ref]);
+
+  const onSelection = (e: any): void => {
+    const active = e.target.textContent;
+    if (!active) {
+      throw Error(`Missing font family in clicked font button`);
+    }
+    fontManager.setActiveFont(active);
+    toggleExpanded();
+  };
+
+  const toggleExpanded = () => setExpanded(exp => !exp);
+
+  const fonts: Font[] =
+    fontManager && Array.from(fontManager.getFonts().values());
+
+  return (
+    <>
+      <SelectedFont
+        ref={ref}
+        type="button"
+        done={loadingStatus === 'finished'}
+        onClick={toggleExpanded}
+        onKeyPress={toggleExpanded}
+        disabled={loadingStatus === 'loading'}
+      >
+        {loadingStatus === 'loading' ? 'Loading Typefaces' : activeFontFamily}
+      </SelectedFont>
+      {loadingStatus === 'finished' && (
```

Comment:
Maybe this ensures that we don't see the arrow?
```suggestion
      {expanded && loadingStatus === 'finished' && (
```

Reply:
![image](https://user-images.githubusercontent.com/587016/67573922-bd249a80-f739-11e9-8a2e-d0c1146782e0.png)